src/scripts/main.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('table tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const rows = [...tbody.querySelectorAll('tr')];
+let rowsCurrent = rows.length;
+let columnsCurrent = rows[0].childElementCount;
+
+const max = 10;
+const min = 2;
+
+const setButton = (el, state) => (el.disabled = state);
+
+appendRow.addEventListener('click', (e) => {
+  const newRow = rows[1].cloneNode(true);
```

Comment:
You are cloning the second row of the table to create a new row. This assumes that the second row is always present and in the correct format. However, if the table is empty or the second row is somehow different, this will not work as expected. It would be safer to create a new row and cells from scratch, or clone a row that you know is in the correct format (e.g. a template row that is hidden in the DOM).